views/index.html

```diff
@@ -1,85 +1,72 @@
-        <li>I can <b>get</b> /api/books to retrieve an aray of all books containing <code>title</code>, <code>_id</code>, & <code>commentcount</code>.</li>
-        <li>I can <b>get</b> /api/books/{_id} to retrieve a single object of a book containing <code>title</code>, <code>_id</code>, & an array of <code>comments</code> (empty array if no comments present).</li>
-        <li>I can <b>post</b> a <code>comment</code> to /api/books/{_id} to add a comment to a book and returned will be the books object similar to <b>get</b> /api/books/{_id}.</li>
-        <li>I can <b>delete</b> /api/books/{_id} to delete a book from the collection. Returned will be 'delete successful' if successful.</li>
-        <li>If I try to request a book that doesn't exist I will get a 'no book exists' message.</li>
-        <li>I can send a <b>delete</b> request to /api/books to delete all books in the database. Returned will be 'complete delete successful' if successful.</li>
-        <li>All 6 functional tests required are complete and passing.</li>
-      </ol>
-      <br>
-      <img src='https://cdn.gomix.com/d7932c52-287f-4dae-b175-631fef453000%2FScreen%20Shot%202016-12-16%20at%201.35.56%20AM.png' />
-    </div>
-    <hr style='margin: 50px'>
-    <div id ='sampleposting'>
-      <h2 style="text-align: left">Test API responses:</h2>
-      <form action="/api/books" method="post" class="border">
-        <h4>Test post to /api/books</h4>
-        Book Title:<br>
-        <input type="text" name="title" value=""><br>
-        <input type="submit" value="Submit">
-      </form>
-      <form action="" method="post" id="commentTest" class="border">
-        <h4>Test post to /api/books/{bookid}</h4>
-        BookId to comment on:<br>
-        <input type="text" name="id" value="" id="idinputtest"><br>
-        Comment:<br>
-        <input type="text" name="comment" value=""><br>
-        <input type="submit" value="Submit">
-      </form>
-    </div>
-    <hr style='margin: 50px'>
-    <div id='sampleui'>
-      <h2 style="text-align: left">Sample Front-End:</h2>
-      <form id="newBookForm" class="border">
-        <input type="text" id="bookTitleToAdd" name="title" placeholder="New Book Title" style="width: 295px">
-        <button type="submit" value="Submit" id="newBook">Submit New Book!</button>
-      </form>
-      <div id='display'></div>          
-      <div id='bookDetail' class='border'>
-        <p id='detailTitle'>Select a book to see it's details and comments</p>
-        <ol id='detailComments'></ol>
-      </div>
-      <button id='deleteAllBooks'>Delete all books...</button>
-    </div>
-    <hr style='margin: 50px'>
+<head>
+  <title>Quality Assurance Project - Personal Library</title>
+  <meta name="description" content="A cool thing made with HyperDev">
+  <link id="favicon" rel="icon" href="https://hyperdev.com/favicon-app.ico" type="image/x-icon">
```

Comment:
The meta description can be removed and we can use the indigo fCC favicon served from our CDN:

`<link rel="icon" type="image/png" href="https://cdn.freecodecamp.org/universal/favicons/favicon-16x16.png" />`